metric_config_parser/metric.py

```diff
@@ -23,13 +24,6 @@
 from .util import converter
 
 
-class AnalysisBasis(Enum):
-    """Determines what the population used for the analysis will be based on."""
-
-    ENROLLMENTS = "enrollments"
-    EXPOSURES = "exposures"
-
-
```

Comment:
We used those in jetstream so I don't think we can simply remove them: https://github.com/mozilla/jetstream/blob/edc883350cab9db3f9ee5992fc6654c979f84f2e/jetstream/analysis.py#L19

Reply:
nvm https://github.com/mozilla/jetstream/pull/1811 will take care of this